tutorials/basic_1_square/code/pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: code
+description: Example code for the the tutorial (Basic Rendering a simple square on the screen)
+
+
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
+  flame: 1.0.0-rc8
+  flutter:
+    sdk: flutter
+
+
+  cupertino_icons: ^1.0.2
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+
+flutter:
+  uses-material-design: true
```

Comment:
do we need the icons? (both)

Reply:
We better keep this, this, is not related to icons, but to Material itself. We had problems in the past by removing this and having silent crashes, so I wouldn't remove this as it does no harm to us

Reply:
it bundles unnecessary things that most games don't need, so I would suggest removing unless needed

Reply:
I don't if that is true, any game that is bundled into a MaterialApp widget will need this, I don't think we should encourage the removal of this line because that could lead people to confusion?

Anyhow, this is just a tutorial and I don't think that removing this line will have a big impact, so if you really want me to remove this, I can remove, let me know.

Reply:
> I don't if that is true, any game that is bundled into a MaterialApp widget will need this

that is not true, that line specifically adds the icons library, and it's there precisely to avoid importing extra stuff if not needed, AFAIK. most game apps don't use material icons (much less cupertino ones).